src/styles/components/_component-card.scss

```diff
@@ -13,10 +13,12 @@
     transition: all 150ms ease-in-out;
     border: 2px solid var(--foreground-color);
 
-    img {
+    &__image {
+      box-sizing: border-box;
       transition: all 150ms ease-in-out;
       width: 100%;
-      height: auto;
+      height: 70%;
+      padding: min(15%, 2rem);
       margin: auto;
     }
 
```

Comment:
Giving the image in the component card a height of 70% and the header section below it a height of 30% was my way of keeping the two sections separate in their container (instead of overlapping as they did before).